resources/definitions/fdmprinter.def.json

```diff
@@ -4919,6 +4919,17 @@
                     "enabled": "magic_spiralize",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "relative_extrusion":
+                {
+                    "label": "Use Relative Extrusion",
+                    "description": "Use relative extrusion rather than absolute extrusion. Before, Cura always used absolute extrusion. Most printers can cope with either relative or absolute extrusion.",
```

Comment:
The description of the setting shouldn't contain info on what old behavior Cura previously had.

I suggest: `Use relative extrusion rather than absolute extrusion. Using relative E-steps makes for easier post-processing of the Gcode. However, it's not supported by all printers and it could lead to small differences in the amount of material being extruded for each line.`